charts/internal/shoot-storageclasses/templates/storageclasses.yaml

```diff
@@ -22,4 +22,14 @@ provisioner: pd.csi.storage.gke.io
 parameters:
   type: pd-ssd
 volumeBindingMode: WaitForFirstConsumer
+
+---
+apiVersion: snapshot.storage.k8s.io/v1beta1
+kind: VolumeSnapshotClass
+metadata:
+  annotations:
+    snapshot.storage.kubernetes.io/is-default-class: "true"
```

Comment:
Should we also add the `resources.gardener.cloud/delete-on-invalid-update=true` annotation? (ref https://github.com/gardener/gardener-resource-manager/pull/69)
/cc @rfranzke 
See https://github.com/gardener/gardener-extension-provider-aws/pull/146#discussion_r459249421

Reply:
Obsolete, see https://github.com/gardener/gardener-extension-provider-aws/pull/146#discussion_r459256870